tests/sentry/api/endpoints/test_organization_user_issues.py

```diff
@@ -86,5 +90,5 @@ def test_has_access(self):
         # now result should include results from team2/project2
         assert response.status_code == 200
         assert len(response.data) == 2
-        assert response.data[0]["id"] == six.text_type(self.group2.id)
+        assert response.data[0]["id"] == six.text_type(self.group3.id)
```

Comment:
why did this work before ?

Reply:
previous test only actually created 2 groups but more tags, test is more realistic now